README.md

```diff
@@ -1,7 +1,7 @@
 GitHub Bot
 ==========
 
-This bot receives commit data from GitHub, and posts in the Showdown chat.
+This bot receives commit data from GitHub, and posts in the Showdown chat, with the option of a Discord server.
```

Comment:
```suggestion
This bot receives commit data from GitHub and can be configured to post in Showdown chat and/or a Discord server.
```
I think in theory the bot should also support Discord only, but thats perhaps out of scope for the time being.

Reply:
👍